asapdiscovery-ml/asapdiscovery/ml/weights/weights.py

```diff
@@ -0,0 +1,140 @@
+import logging
+from pathlib import Path
+from typing import List, Union  # noqa: F401
+
+import requests
+import validators
+import yaml
+
+
+def fetch_weights_from_spec(
+    yamlfile: str,
+    models: Union[list[str], str],
+    local_path: str = "./_weights/",
+    force_fetch: bool = False,
+):
```

Comment:
add return signature here?
```suggestion
) -> dict:
```

Reply:
Good spotting but I will use `Dict[str, Path]`